web/app/models/ability.rb

```diff
@@ -0,0 +1,32 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+      user ||= User.new # guest user (not logged in)
+      if user.role == 'admin';
+        can :manage, :all
+      else
+        can :read, :all
+      end
+    #
+    # The first argument to `can` is the action you are giving the user 
```

Comment:
Do you like to keep comments around? I'm not a fan personally unless it's a docstring type thing. Up to you.